site/README.md

```
@@ -0,0 +1,31 @@
+## Develop
+
+```sh
+bun install
+bun run dev      # http://localhost:4321
+bun run build    # outputs to dist/
+bun run preview  # serve the production build locally
+```
+
+## Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)